akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -214,12 +214,14 @@ object IntroSpec {
 
   //#chatroom-main
   object Main {
+    import ChatRoom._
+
     def apply(): Behavior[NotUsed] =
       Behaviors.setup { context =>
         val chatRoom = context.spawn(ChatRoom(), "chatroom")
         val gabblerRef = context.spawn(Gabbler(), "gabbler")
         context.watch(gabblerRef)
-        chatRoom ! ChatRoom.GetSession("ol’ Gabbler", gabblerRef)
+        chatRoom ! GetSession("ol’ Gabbler", gabblerRef)
```

Comment:
Prefixing the actor messages with the name of the actor to make it clear which actor protocol is used was intentional, so fixing this will rather be about making sure the surrounding `ChatRoom` companion object is visible where the message definitions are shown in the docs.

Reply:
Okay, Right.
 I fixed the docs about  the protocol definition is defined inside of the ChatRoom companion object.
how about this change?

<img width="1096" alt="スクリーンショット 2021-02-17 0 01 48" src="https://user-images.githubusercontent.com/21272619/108082755-c3205680-70b5-11eb-8ab0-064dab7db2d4.png">